src/App.jsx

```diff
@@ -1,20 +1,25 @@
 // LIBRARY IMPORTS
-import React from 'react';
+import React, { useState, useRef } from 'react';
 import {
 	BrowserRouter as Router,
 	Routes,
 	Route,
 } from 'react-router-dom';
 
 // LOCAL IMPORTS
-
+import { bioData } from './assets/bioData.js';
 import { Home, Layout, ProfessionalsPage } from './views';
 
 function App() {
+  // SET STATES
+  const [data, setData] = useState(bioData);
+
```

Comment:
since the data needs to be changed in the search and filter functions I've lifted its import up to `App.jsx` and set a state with it so that it can be changed with `setState`